spring-cloud-gcp-pubsub-stream-binder/src/main/java/org/springframework/cloud/gcp/stream/binder/pubsub/provisioning/PubSubChannelProvisioner.java

```diff
@@ -47,65 +47,61 @@
 
 	private final PubSubAdmin pubSubAdmin;
 
-	private final Set<String> anonymousSubscriptions = new HashSet<>();
+	private final Set<String> anonymousSubscriptionNames = new HashSet<>();
```

Comment:
Please consider renaming this. "anonymous" literally means without-name, so "anonymous subscription names" is like "subscription names without names".

Reply:
`anonymousGroupSubscriptionNames` ?